packages/recomponents/src/components/r-date-range/r-date-range.vue

```diff
@@ -339,6 +345,9 @@
                     this.close();
                 });
             },
+            checkDatesAreSame(date1, date2) {
+                return moment(date1).format(DateTimeFormats.datePickerDate) === moment(date2).format(DateTimeFormats.datePickerDate);
```

Comment:
We should use moment's built in equivalency check https://momentjscom.readthedocs.io/en/latest/moment/05-query/02-is-same/

Unless there is a reason for avoiding it.

Reply:
@mattsanf oh! thanks!!! i've updated the method 👍